docs/rfc/rfc-1.adoc

```diff
@@ -825,17 +880,48 @@ Alphabetized list of Governable Parameters with additional notes.
   be considered for a sweep.
 - `failed_heartbeat_reward_removal_period`: The amount of time an operator is
   removed from reward eligibility after failing a heartbeat.
+- `fraud_challenge_defend_timeout`: The amount of time the wallet has to defend
+  against a fraud challenge.
+- `fraud_challenge_deposit_amount`: The amount of ETH the party challenging the
+  wallet for fraud needs to deposit.
+- `fraud_notifier_reward_multiplier`: The percentage of the notifier reward
+  from the staking contract the notifier of a fraud receives.
+- `fraud_slashing_amount`: The amount of stake slashed from each member of a
+  wallet for a fraud.
+- `heartbeat_block_length`: The number of ethereum blocks until the next
+  heartbeat. If set to 40, then the signers sign every 40th block.
 - `heartbeat`: The number of group members required for a
   <<heartbeat,heartbeat>> to be successful.
-- `heartbeat_block_length`: The number of ethereum blocks until the next heartbeat.
-  If set to 40, then the signers sign every 40th block.
 - `max_gas_refund_price`: The highest amount of gwei that the gas refund
   contract will pay out per gas for a refund transaction.
+- `random_transfer_depth`: The depth (sorted youngest to oldest) at which we
+  randomly select a new wallet to be the custodian of the old wallet's funds
+  when closing a wallet. For instance, a depth of 3 means that the 3 youngest
+  wallets can be randomly chosen.
+- `redemption_request_timeout_redemeer_bonus_multiplier`: The percentage of the
+  notifier reward from the staking contract the redeemer receives in case of a
+  redemption timeout.
+- `redemption_request_timeout_redemeer_bonus_multiplier`: The percentage of the
+  notifier reward from the staking contract the redeemer receives in case of a
+  redemption timeout.
+- `redemption_request_timeout_slashing_amount`: The amount of stake slashed
+  from each member of a wallet for a timed-out redemption request.
+- `redemption_request_timeout_slashing_amount`: The amount of stake slashed
+  from each member of a wallet for a timed-out redemption request.
+- `redemption_request_timeout`: The amount of time the wallet has to provide
+  redemption proof.
+- `redemption_request_timeout`: The amount of time the wallet has to provide
+  redemption proof.
+- `redemption_treasury_fee`: The percentage of redeemed amount put aside as a
+  treasury fee.
+- `redemption_treasury_fee`: The percentage of redeemed amount put aside as a
+  treasury fee.
```

Comment:
Some parameters got duplicated
```suggestion
- `redemption_request_timeout_redemeer_bonus_multiplier`: The percentage of the
  notifier reward from the staking contract the redeemer receives in case of a
  redemption timeout.
- `redemption_request_timeout_slashing_amount`: The amount of stake slashed
  from each member of a wallet for a timed-out redemption request.
- `redemption_request_timeout`: The amount of time the wallet has to provide
  redemption proof.
- `redemption_treasury_fee`: The percentage of redeemed amount put aside as a
  treasury fee.
```